pkg/kubectl/cmd/util/helpers.go

```diff
@@ -46,6 +46,7 @@ import (
 
 const (
 	ApplyAnnotationsFlag = "save-config"
+	AddOnLabel           = "kubernetes.io/cluster-service"
```

Comment:
So since `kubernetes.io/cluster-service=true` already exists and it's hard to get people switch to a generic annotation, I guess we can use both annotations here.